src/main/java/subSystems/Arm.java

```diff
@@ -0,0 +1,65 @@
+package subSystems;
+
+import com.flash3388.flashlib.robot.RunningRobot;
+import com.flash3388.flashlib.scheduling.Subsystem;
+import com.flash3388.flashlib.time.Time;
+import com.revrobotics.CANSparkMax;
+import com.flash3388.flashlib.robot.control.PidController;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class Arm extends Subsystem {
+
+    private double angle2Target;
+    private CANSparkMax master;
+    private CANSparkMax follower;
+    private DutyCycleEncoder encoder;
+    private PidController pid;
+    private static final double KP = 0;
+    private static final double KI = 0;
+    private static final double KD = 0;
+    private static final double KF = 0;
+    private static final double ERROR = 1;
+    private static final double LIMIT = 1;
+
+
+    public Arm(double angle2Target, CANSparkMax master, CANSparkMax follower, DutyCycleEncoder encoder){
+        this.angle2Target = angle2Target;
+        this.master = master;
+        this.follower = follower;
+        this.encoder = encoder;
+
+        follower.follow(master);
+
+        SmartDashboard.putNumber("KP", KP);
```

Comment:
this is not necessary when using `PidController.newNamedController`, it is already done automatically.